### ISSUE TITLE
Meaning of missing tokenBinding in ClientCollectedData?

### ISSUE THREAD

**Issue by dwaite** [w3c/webauthn#907]
the tokenBinding element is not required in ClientCollectedData. Is this meant to be a required value, or is perhaps the meaning when missing equivalent to "not-supported" or some other implicit status "unspecified"?

**Comment by emlun** [comment]
I would interpret a missing `tokenBinding` to mean an "unspecified" or "unknown" state, but the meaning may indeed to be explicitly stated in the spec.

@agl You wrote this - what's your take on how to interpret a missing `tokenBinding` value?

**Comment by arnar** [comment]
I think @kpaulh wrote the last revision.

**Comment by agl** [comment]
Agreed, I think a missing value would have to mean that TB is not supported.

**Comment by emlun** [comment]
Yes, in that case it sounds to me like `tokenBinding` should default to `{ "status": "not-supported" }`, no? I'm not sure whether that means it should also be required, though.

**Comment by kpaulh** [comment]
That's actually what Chrome does right now (rather than omit it entirely), but similarly not sure that it should be required.

**Comment by selfissued** [comment]
Leaving it blank seems good to me.

**Comment by agl** [comment]
I'm going to make a PR to address this by removing the &ldquo;not-supported&rdquo; value. (I'll also align Chrome with Firefox's behaviour here.)

**Comment by equalsJeffH** [comment]
agreed, PR approved, thx!
